components/org.wso2.carbon.identity.application.authenticator.adapter/src/main/java/org/wso2/carbon/identity/application/authenticator/adapter/AuthenticationResponseProcessor.java

```diff
@@ -152,5 +158,25 @@ public ActionExecutionStatus<Error> processErrorResponse(Map<String, Object> eve
 
         return new ErrorStatus(new Error(errorResponse.getErrorMessage(), errorResponse.getErrorDescription()));
     }
+
+    private static String addFlowId(String url, String flowId) throws ActionExecutionResponseProcessorException {
```

Comment:
Let's remove this logic and let the extension developer to decide what sort of state parameter they should add.
As flowId is communicated they can either use it or have a different state parameter